### DISCUSSION TITLE
Coordinate Reference System is not defined

### DISCUSSION THREAD

**Discussion by AfxentiouNikolas** [iTwin/community#15] [Q&A]
Hello! After the new iTwin.js 4.0 Release, I am facing this issue. I am creating a new iModel using iTwin synchronizer, the same methodology I used previously and for a reason my building is elevated with wrong coordinates. I am using Revit 2023 and adding the correct coordinates for the building. Any ideas?
On iTwin platform write the following error: 
Coordinate Reference System is not defined
This can result in iModel spatial alignment issues. Please follow our [best practices guides](https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/67080/itwin-spatial-alignment) to validate the spatial positioning and make any necessary adjustments before including engineering data from various formats.
![iModel](https://github.com/iTwin/community/assets/97607951/4cd36870-5b7d-4d43-93f8-cff58176d2be)

**Comment by aruniverse** [comment]
Did you go through the [best practices guides](https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/67080/itwin-spatial-alignment) ?
